java-client-model/src/main/java/com/cumulocity/rest/representation/inventory/InventoryRepresentation.java

```diff
@@ -0,0 +1,89 @@
+package com.cumulocity.rest.representation.inventory;
+
+import com.cumulocity.rest.representation.AbstractExtensibleRepresentation;
+import org.svenson.JSONProperty;
+
+/**
+ * A Java Representation for the MediaType Inventory, making a compromise between
+ * https://startups.jira.com/wiki/display/MTM/Inventory and the example in 
+ * https://startups.jira.com/wiki/display/MTM/Requests+to+Inventory
+ */
```

Comment:
this comment can be refreshed or removed

Reply:
TODO